src/components/Navbar/hooks.ts

```diff
@@ -0,0 +1,18 @@
+import { useRef } from "react";
```

Comment:
todo: The name should be "Navbar.hooks.ts"

Reply:
makes sense! :ok_hand: